lib/rules/extends-correct-class.js

```diff
@@ -15,12 +15,33 @@ function getExtendsOption(node) {
   return null
 }
 
+/**
+ * Format the array of allowed super names for a given class
+ * into a string for reporting.
+ *
+ * @param {Array<string>} allowedSuperNames
+ * @example
+ * formatNames(['HTMLElement']) // 'HTMLElement'
+ * formatNames(['HTMLElement', 'CustomElement']) // '{HTMLElement, CustomElement}'
```

Comment:
Tried this out locally and I think the comment should be:

```suggestion
 * formatNames(['HTMLElement', 'CustomElement']) // HTMLElement or CustomElement
```